analytics_data_api/v0/views/learners.py

```diff
@@ -36,24 +36,33 @@ class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
 
     **Response Values**
 
-        Returns the learner metadata and engagement data:
+        Returns metadata and engagement data for the learner in JSON format.
 
             * username: User's username.
-            * enrollment_mode: Enrollment mode (e.g. "honor).
+            * enrollment_mode: The user's learning track (for example, "audit"
+              or "verified").
```

Comment:
we no longer offer honor code certificates, so this seemed more likely to be a useful example...